### DISCUSSION TITLE
Service MySQL can not start

### DISCUSSION THREAD

**Discussion by Grandi0z** [leokhoa/laragon#757] [Q&A]
I used Laragon yesterday everything was fine, 
I didn't do any particular configuration, but since this morning, here's the error I'm getting.

<img width="508" alt="laragon" src="https://github.com/leokhoa/laragon/assets/114248080/0809e9fe-1c2b-4eaa-9f2f-1c030fd63f98">

And the message in my "mysqld.log":

<img width="914" alt="laragon2" src="https://github.com/leokhoa/laragon/assets/114248080/1f5d9610-c97b-4467-9809-fc0c9954134b">

I've already checked that there are no other processes on port "3306". I've even tried to change the port, but I get the same error. 

I've also uninstalled Laragon, deleted the "laragon" folder and reinstalled the application like suggested in this [discussion .](https://github.com/leokhoa/laragon/discussions/712) But the error persists

**Comment by Grandi0z** [comment]
I don't know what's happening, yesterday I gave up, and I decided to work on another project  ( react project ) until I find a solution for my Laragon, this morning I opened my PHP project, (the one I was supposed to work on yesterday) I open Laragon, and it works normally.

**Reply by DamChtlv** [reply]
Hello @Grandi0z 
Same as https://github.com/leokhoa/laragon/discussions/712

I really suspect this error to show when we shutdown our PC and Laragon is still up / on
_(which might mess with some temp files being incorrectly saved)_
